.github/workflows/ci.yml

```
@@ -9,11 +9,11 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: set up python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
         with:
-          python-version: 3.9
+          python-version: 3.10
```

This needs to be quoated, otherwise it's interpreted as `3.1`

```suggestion
          python-version: "3.10"
```

---

Wow GitHub, wow.

---

Got it, I was able to find the issue as well: https://github.com/actions/setup-python/issues/160

---

The number of references to that issue... yikes.

---

Eh, it's how YAML parses it, not GitHub.

---

YAML is just a writing style, I would hope GitHub has a custom YAML parser on its end that could handle this.